.github/workflows/ci.yml

```diff
@@ -19,6 +19,7 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: 12.x
+          cache: npm
 
       - name: get yarn cache dir
```

Comment:
as you can see below we already have caching set up here, so if we migrate to the new way of caching things we should remove the old one